frontend/src/scenes/onboarding/sdks/feature-flags/FeatureFlagsSDKInstructions.tsx

```diff
@@ -18,17 +19,18 @@ import {
 
 export const FeatureFlagsSDKInstructions: SDKInstructionsMap = {
     [SDKKey.JS_WEB]: FeatureFlagsJSWebInstructions,
-    [SDKKey.REACT]: FeatureFlagsReactInstructions,
-    [SDKKey.NEXT_JS]: FeatureFlagsNextJSInstructions,
-    [SDKKey.IOS]: FeatureFlagsIOSInstructions,
-    [SDKKey.REACT_NATIVE]: FeatureFlagsRNInstructions,
+    [SDKKey.ANGULAR]: FeatureFlagsAngularInstructions,
     [SDKKey.ANDROID]: FeatureFlagsAndroidInstructions,
+    [SDKKey.API]: FeatureFlagsAPIInstructions,
     [SDKKey.FLUTTER]: FeatureFlagsFlutterInstructions,
+    [SDKKey.GO]: FeatureFlagsGoInstructions,
+    [SDKKey.IOS]: FeatureFlagsIOSInstructions,
+    [SDKKey.NEXT_JS]: FeatureFlagsNextJSInstructions,
     [SDKKey.NODE_JS]: FeatureFlagsNodeInstructions,
+    [SDKKey.PHP]: FeatureFlagsPHPInstructions,
     [SDKKey.PYTHON]: FeatureFlagsPythonInstructions,
+    [SDKKey.REACT]: FeatureFlagsReactInstructions,
+    [SDKKey.REACT_NATIVE]: FeatureFlagsRNInstructions,
     [SDKKey.RUBY]: FeatureFlagsRubyInstructions,
-    [SDKKey.PHP]: FeatureFlagsPHPInstructions,
-    [SDKKey.GO]: FeatureFlagsGoInstructions,
-    [SDKKey.API]: FeatureFlagsAPIInstructions,
```

Comment:
I've reordeded the list of frameworks in the list to be alphabetical (but keeping JS Web at the top). It'll be easier for people to find their framework this way